content-services/content-sources-backend
# Fixes 2582,2425: add account_id to task table, trim error

## Summary

Adds account_id to tasks table.  This makes querying by account_id easier and faster.
Note that after this change, any existing tasks will have a null account_id, but since we are still in stage, that is okay.

Change error length to 4000 chars, and auto trim error if its longer than that

I did these in a single PR as they would normally conflict

## Testing steps

1. Deploy this pr and migrate the db
2. Snapshot some repository
3. Use the admin tasks interface
4. filter by account id

Verify nothing is broken, behavior should be the same

Unsure how to test the error length, i wasn't able to reproduce it but the unit test should prove that its solved
